### PR TITLE
Moved Onboarding Screens to After User Registration

### LINKED ISSUE #156: Move onboarding screens to be after the registration flow
For a better user experience, let's have our users view the onboarding information after they register an account so that they're more likely to retain it.

Specs:
- [x] Participants are only shown onboarding screens after they have registered an account
- [x] Participants are only shown the onboarding screens once (after registration)

### PR DESCRIPTION
**Pull request by shashjar**
Onboarding screen flow is now shown to a user after they register for an account, not before. Shown only after the user clicks continue from the email verification page, and then routes the user to the home page. User can view onboarding screens again through the Help button the home page, and will again be routed back to the home page afterward.

### CODE REVIEW

File: frontend/App.tsx [hunk 1]
```diff
@@ -57,7 +55,7 @@ export default function App() {
   }
 
   useEffect(() => {
-    AsyncStorage.getItem(HAS_SEEN_ONBOARDING).then(val => setStartingState(val ? "SignInFlow" : "Onboarding"));
+    setStartingState("SignInFlow");
```
Comment (emaela) [inline frontend/App.tsx]: I don't think we need startingState anymore - double check me, but if that's true then we can just remove it.

Comment (shashjar) [reply frontend/App.tsx]: removed - initial route is now set to "SignInFlow" for navigation

**Review by emaela — APPROVED** [review]
LGTM

**Review by emaela — APPROVED** [review]
LGTM

**Pull request merged**